lib/components/map/connected-transitive-overlay.js

```diff
@@ -7,6 +7,12 @@ import { getTransitiveData } from '../../util/state'
 
 const mapStateToProps = (state, ownProps) => {
   const { labeledModes, styles } = state.otp.config.map.transitive || {}
+
+  // If trip viewer is not active, do not show trip on map
```

Comment:
Comment seems incorrect (see connected-endpoints-overlay).

Reply:
Comments clarified in 3ef87bd0ff20ff07bd30631d89827dc8dada9672